pages/dkp/kommander/2.2/install/air-gapped/catalog/index.md

```diff
@@ -1,110 +1,254 @@
 ---
 layout: layout.pug
-navigationTitle: Host on Gitea
-title: Host on Gitea
+navigationTitle: Install Kommander in an air-gapped environment with catalog applications
+title: Install Kommander in an air-gapped environment with catalog applications
 menuWeight: 10
-excerpt: How to host git repositories on Gitea for air gapped deployments
+excerpt: Install Kommander in an air-gapped environment with catalog applications
+beta: false
+enterprise: true
 ---
 
-DKP supports using an external catalog git repository to install applications. In cases where an external repository is not accessible from within the cluster, to install applications, use the Gitea server that comes out-of-box with a Kommander deployment.
+
+This topic shows how to run Kommander on top of an [air-gapped Konvoy cluster][air-gap-konvoy] installation with catalog applications.
 
 ## Prerequisites
 
-- Install `git` in the local environment (from where a connection to air-gapped cluster is established).
+Before installing, ensure you have:
 
-## Host external repository on Gitea
+-   A Docker registry containing all the necessary Docker images, including the Kommander and [DKP catalog application][dkp_catalog_applications] images.
 
-To configure the Gitea server, follow these steps:
+-   A charts bundle file containing all Helm charts that Kommander installation needs, including the [DKP catalog applications][dkp_catalog_applications] charts.
 
-1.  Set the `VERSION` environment variable to the version of Kommander you want to install, for example:
+-   Connectivity with clusters attaching to the management cluster:
```

Comment:
Do we need to mention this topic here somehow?

https://docs.d2iq.com/dkp/kommander/2.1/clusters/attach-cluster/cluster-with-network-restrictions/

Reply:
seems like a good idea! added in a09edf2b18c9e31f3c3be09e82f6a5002baf9fdb